aws/registers/templates/fluentd.conf

```diff
@@ -52,4 +52,13 @@
     endpoint https://endpoint1.collection.eu.sumologic.com/receiver/v1/http/${sumologic_key}
     log_format json
   </store>
+
+  <store>
+    type logit
+    stack_id ${logit_stack_id}
+    port 16081
```

Comment:
is it worth making the port number configurable too? (these do vary with stack id)